config/v1/types_infrastructure.go

```diff
@@ -354,8 +354,21 @@ type AWSPlatformSpec struct {
 	// There must be only one ServiceEndpoint for a service.
 	// +optional
 	ServiceEndpoints []AWSServiceEndpoint `json:"serviceEndpoints,omitempty"`
+
+	// LBType allows user to set a load balancer type.
```

Comment:
You need to explain what happens when this field isn't set. You also need to explain what would happen if the field is changed (if that is allowed) while the cluster is running.

Reply:
Reading the enhancement, this seems it's the default but can be overriden right? That will need to be explained in this godoc

Reply:
Hi @JoelSpeed ! I have added the description for what happens when this value is set and not set.

Reply:
@JoelSpeed I have added the description as to what will happen when the value is set and not set.

	// When this field is set the default ingresscontroller will get created using the specified LBType.
	// If this field is not set then the default ingress controller of LBType Classic will be created.